extensions/wikia/UserLogin/tests/UserLoginTest.php

```diff
@@ -33,60 +33,6 @@ protected function setUpMock() {
 
 	/**
 	 * @group Slow
-	 * @slowExecutionTime 0.44648 ms
```

Comment:
UserLogin/Signup tests are notorious for spewing crap in logs and being slow. It may be worth it to disable the rest of the tests in this suite (`@group Broken` or `$this->markTestskipped()`), since legacy signup is no longer enabled on production.